src/modules/hunting/cvehunter.py

```diff
@@ -37,26 +37,20 @@ class IsVulnerableToCVEAttack(Hunter):
     def __init__(self, event):
         self.event = event
         self.headers = dict()
+        # From within a Pod we may have extra credentials
+        if self.event.auth_token:
+            self.headers = {'Authorization': 'Bearer ' + self.event.auth_token}
         self.path = "https://{}:{}".format(self.event.host, self.event.port)
         self.service_account_token_evidence = ''
```

Comment:
I think we no longer use this self.service_account_token_evidence now and could get rid of it

Reply:
@lizrice  You are right. Removed that line. Please merge.